pixeltable/exec/globals.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import dataclasses
+
+from pixeltable.exprs import ArrayMd
+
+INLINED_OBJECT_MD_KEY = '__pxtinlinedobjmd__'
+
+
+@dataclasses.dataclass
+class InlinedObjectMd:
```

Comment:
I'm curious why this doesn't live in the same place as `ArrayMd`? (what's the reasoning behind putting them in different places?)